master/master.cfg

```diff
@@ -51,7 +51,7 @@ c['titleURL'] = "http://" + gerrit_url
 c['buildbotURL'] = "http://%s:%s" % (bb_master_url, bb_web_port)
 
 ####### FACTORIES
-build_factory = createBuildFactory(gerrit_repo_http)
```

Comment:
This leaves createBuildFactory() as dead code.  That function needs to be removed in this patch.